tests/testthat/test-db.R

```diff
@@ -103,10 +103,6 @@ test_that("avoid unserialisable parameters", {
   skip_on_cran_windows()
   path <- prepare_orderly_example("parameters", testing = TRUE)
   t <- Sys.Date()
-  id <- orderly_run("example", parameters = list(a = t, b = TRUE, c = "one"),
```

Comment:
This is now validated earlier, and the remaining tests never relied on a report being run